### ISSUE TITLE
Unit tests needs to run in the CI

### ISSUE THREAD

**Issue by nleach999** [checkmarx-ts/CxAnalytix#14]
### Description

The unit tests are currently not part of the CI due to some incompatibility in the build docker image.  It looks like the unit tests need to run under .Net core 3.1.

### Expected Behavior

Unit tests should execute successfully in the build.

### Actual Behavior

The build times out because the unit test execution appears to try to download many unnecessary dependencies.

### Reproduction

Enable unit tests in the CI pipeline, execute a build.

### Environment Details

N/A

**Comment by nleach999** [comment]
I've done some preliminary work on this, commit is just waiting on more testing when I have time.

**Comment by nleach999** [comment]
fixed with commit d12de892ca36c52489d0252378f9b0579baac981
